.travis.yml

```diff
@@ -34,6 +34,9 @@ before_install:
 - nvm install $NODE_VERSION
 - nvm use $NODE_VERSION
 - npm install -g greenkeeper-lockfile@1
+- sudo apt-get remove -q 'postgresql-*'
+- sudo apt-get update -q
+- sudo apt-get install -q postgresql-11 postgresql-11-postgis-2.5 postgresql-11-postgis-2.5-scripts
```

Comment:
We can only install this for PG build to improve speed. If we can’t get it to work we can downgrade to 10